config.js

```diff
@@ -99,32 +99,32 @@ config.iota = {
      * Configuration for the IoT Manager. If the IoT Agent is part of a configuration composed of multiple IoTAgents
      * coordinated by an IoT Manager, this section defines the information that will be used to connect with that manager.
      */
-    iotManager: {
+    //iotManager: {
```

Comment:
Yes, I do. This is the default config file, and having the IoTManager section uncommented leads to a connection error when the IOTAgent is started, as no one is going to have an IoTManager installed in its own laptop.